test/test-real-values.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+const path = require('path');
+const compareVersions = require('compare-versions');
+const chalk = require('chalk');
+
+/** @type {Record<string, string[]>} */
+const browsers = {
+  api: [],
+  css: ['ie'],
+  html: [],
+  http: [],
+  svg: [],
+  javascript: [],
+  mathml: [],
+  webdriver: [],
+  webextensions: [],
+  xpath: [],
+  xslt: []
+};
+
+/**
+ * @param {SupportBlock} supportData
+ * @param {{error:function(...unknown):void}} logger
+ */
+function checkRealValues(supportData, browsers, logger) {
+  let hasErrors = false;
+  const browsersToCheck = Object.keys(supportData);
```

Comment:
This will only consider browsers that we have listed in the JSON file. However, our schema allows to omit browsers. So, say you want to only allow real values for `samsunginternet_android`, it would currently pass this test in the `mathml/` folder, although we don't have samsung in there at all. So, the `null` test is always testing if literally `null` is written out, or if the browser is not existent in the data at all.

Reply:
Ah, that makes sense.  Lemme see what I can do, in that case!